hurumap_global/templates/homepage.html

```diff
@@ -81,7 +81,7 @@ <h1 class="text-center">About {{ HURUMAP.name }}</h1>
         $('#locate-me').html('Locating... <i class="fa fa-location-arrow fa-pulse"></i>');
 
         navigator.geolocation.getCurrentPosition(function(position) {
-            var url = 'https://maps.googleapis.com/maps/api/geocode/json?sensor=false&language=en&latlng=' + position.coords.latitude + ',' + position.coords.longitude;
+            var url = 'https://maps.googleapis.com/maps/api/geocode/json?sensor=false&language=en&latlng=' + position.coords.latitude + ',' + position.coords.longitude + '&key=AIzaSyDbs8VaY2_tyvDAoOQN1keRefIaEzH1w8k';
```

Comment:
Where did this key come from & how are we managing it @phillipahereza?

Reply:
@kilemensi from CfA Google APIs dashboard, I'll restrict it only accept pings from https://hurumap.org after this has been deployed